src/servers/ZoneServer2016/entities/character.ts

```diff
@@ -851,6 +877,40 @@ export class Character2016 extends BaseFullCharacter {
       : undefined;
   }
 
+  pGetItemData(server: ZoneServer2016, item: BaseItem, containerDefId: number) {
+    let durability: number = 0;
+    switch (true) {
```

Comment:
Wow that's a strange tricks that i didn't knew, there is no better way of doing this?

Reply:
I mean we could do a bunch of if/elses but it would look bad